From acbee4edacb80b5eeaff2480712fe98e56443997 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 19 Jan 2016 18:18:52 -0800
Subject: [PATCH] include sys/types.h for mode_t

mode_t is used in target.h, so we need to include sys/types.h to get the
defintion

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Pending

 gdb/gdbserver/target.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gdb/gdbserver/target.h b/gdb/gdbserver/target.h
index 9a40867..aaecab9 100644
--- a/gdb/gdbserver/target.h
+++ b/gdb/gdbserver/target.h
@@ -27,6 +27,7 @@
 #include "target/waitstatus.h"
 #include "mem-break.h"
 #include "btrace-common.h"
+#include <sys/types.h>
 
 struct emit_ops;
 struct buffer;
-- 
2.7.0

